extensions/smallrye-rest-client/runtime/src/main/java/io/quarkus/smallrye/restclient/runtime/RestClientBuilderImpl.java

```diff
@@ -68,12 +69,16 @@
     public static final ClientHeadersRequestFilter HEADERS_REQUEST_FILTER = new ClientHeadersRequestFilter();
 
     public static boolean SSL_ENABLED = false;
+    public static boolean REGISTER_BUILTIN_PROVIDERS = true;
+    public static String PROVIDERS_TO_REGISTER = "";
 
     RestClientBuilderImpl() {
         ClientBuilder availableBuilder = ClientBuilder.newBuilder();
 
         if (availableBuilder instanceof ResteasyClientBuilder) {
             builderDelegate = (ResteasyClientBuilder) availableBuilder;
+            builderDelegate.property(ResteasyContextParameters.RESTEASY_USE_BUILTIN_PROVIDERS, REGISTER_BUILTIN_PROVIDERS);
+            builderDelegate.property(ResteasyContextParameters.RESTEASY_PROVIDERS, PROVIDERS_TO_REGISTER);
```

Comment:
You don't want to support it here?

Reply:
Not sure what you mean.

Reply:
Well, in the case of `RestClientBuilderImpl`, AFAICS, you use all the built-in providers. Which means that you will need them all to be registered.

Reply:
Setting that property is telling RESTEasy we don't want to use all the built-in providers.

Other than my mess up on the processor, not sure why you see `RestClientBuilderImpl` using all the built-in providers?

Reply:
Ah, yes, it's my mistake. Read it as `public static final` instead of `public static`. This part looks good, sorry for the noise.